internal/operators/mce/mce_operator_test.go

```diff
@@ -93,3 +93,26 @@ var _ = Describe("MCE Operator", func() {
 		)
 	})
 })
+
+var _ = Describe("Get right min disk size", func() {
+	var (
+		fullHaMode = models.ClusterHighAvailabilityModeFull
+		snoMode    = models.ClusterHighAvailabilityModeNone
+	)
+	When("cluster is SNO", func() {
+		cluster := &models.Cluster{HighAvailabilityMode: &snoMode}
+		It("should return the sum of all required PVCs", func() {
+			minDiskSize := GetMinDiskSizeGB(cluster)
+			expectedMinDiskSize := int64(70)
+			Expect(expectedMinDiskSize).To(Equal(minDiskSize))
+		})
+	})
+	When("cluster is HA", func() {
+		cluster := &models.Cluster{HighAvailabilityMode: &fullHaMode}
+		It("should return the maximum value of any required PVCs", func() {
+			minDiskSize := GetMinDiskSizeGB(cluster)
+			expectedMinDiskSize := int64(50)
+			Expect(expectedMinDiskSize).To(Equal(minDiskSize))
+		})
+	})
```

Comment:
This all feels like it could be simplified, maybe something like this?
```suggestion
	It("should return the sum of all required PVCs for a SNO cluster", func() {
	        snoMode := models.ClusterHighAvailabilityModeNone
		minDiskSize := GetMinDiskSizeGB(&models.Cluster{HighAvailabilityMode: &snoMode})
		expectedMinDiskSize := int64(70)
		Expect(expectedMinDiskSize).To(Equal(minDiskSize))
	})
	It("should return the maximum value of any required PVCs for a HA cluster", func() {
	        fullHaMode := models.ClusterHighAvailabilityModeFull
		minDiskSize := GetMinDiskSizeGB(&models.Cluster{HighAvailabilityMode: &fullHaMode})
		expectedMinDiskSize := int64(50)
		Expect(expectedMinDiskSize).To(Equal(minDiskSize))
	})
```

Reply:
Yes, I have the habit to make it human readable: "When cluster is SNO" (or maybe "context cluster is SNO" "it should return..."
I can simplify it as suggested though, no big difference